package.json

```diff
@@ -5,6 +5,7 @@
   "license": "Apache-2.0",
   "dependencies": {
     "@clack/prompts": "^0.7.0",
+    "@coinbase/onchainkit": "^0.19.7",
```

Comment:
I was getting import errors of `@coinbase/onchiankit` being unavailable. Adding it as a dependency to the root package.json fixed my errors.

Reply:
doesn't make much sense adding this here.